volunteer/index.html

```diff
@@ -2,12 +2,40 @@
 enable-mentor-alert: false
 title: "Volunteer Application Form"
 scripts: |
-    <script src="js/spring15android.js"></script>
 ---
+<div class="banner-image volunteer">
+    <h1 class="banner-title">Volunteers</h1>
+</div>
+
+<!--Card-->
+<div class="row">
+  <div class="col-sm-6 col-md-6">
+    <div class="thumbnail volunteer-card" style="margin-right: 5%;">
+      <img src="../images/volunteers/classroomVolunteers.jpg">
+        <div class="caption">
+          <h3>Classroom Volunteer</h3>
+          <p>Classroom Volunteers play an essential role by providing one on one support to our students and assisting our instructors. With an understanding of the fundamentals of HTML, CSS, java, and javascript, you will engage a diverse group of students through a project based coding course. </p>
+        </div>
+    </div>
+  </div>
+
+  <div class="col-sm-6 col-md-6">
+    <div class="thumbnail volunteer-card" style="margin-left: 5%;">
+      <img src="../images/volunteers/volunteerPanel.jpg">
+        <div class="caption">
+          <h3>One-Time Volunteer</h3>
+          <p>One-Time Events Volunteer support the Mission Bit staff with activities throughout the year. Events typically occur outside of the classroom, at various locations throughout San Francisco and Oakland (e.g. Demo Day, Mission Bit Gala, Student Admissions Interview, Workshops). </p>
+        </div>
+    </div>
+  </div>
+</div>
+<!--/.Card-->
 
+<p><a href="#" class="btn btn-lg btn-default shadow hardware" role="button">Learn More</a>
```

Comment:
This should say "sign up", not Learn more.

Reply:
Also, the link just goes to the homepage, so this is missing the form page (which was just the old page, so keep the old page, rename it, and add the path to href)